src/App.js

```diff
@@ -2,32 +2,39 @@ import React, { Component } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Search } from './components/Search';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+    moviesList: moviesFromServer,
+  };
+
+  onFilter = (event) => {
+    const { value, name } = event.target;
+
+    this.setState({
+      [name]: value,
+    });
```

Comment:
```suggestion
    this.setState({
      query: value,
    });
```
также понятнее)

это было бы полезно если бы у тебя было несколько инпутов и их обрабатывала одна функция